apps/web/src/config/constants/priceHelperLps/pools/56.ts

```diff
@@ -1,4 +1,4 @@
-import { getAddress } from 'viem'
+import { isAddress } from 'utils'
```

Comment:
can we rename it to getAddress? From the semantic meaning, it's getting the checksumed address

Reply:
probably need to update the implementation as well. It's quite confusing to return both string and bool.

Can we return empty string or undefined to indicate it's not a valid address?

Reply:
imo would be great to have two separate utility functions:
`isAddress(value: any): value is Address`
`getAddress(value: any): Address | undefined`

Reply:
safeGetAddress?
I think some of the place are meant to be throw error
Casting into false is just hiding the issue

Reply:
Great job! Will take a look